src/views/ExportFile.vue

```diff
@@ -1,34 +1,18 @@
-<template>
-    <div class="container">
-        <small-page v-if="$route.name === `export-file-success`">
-            <router-view/>
-        </small-page>
-        <!-- <a class="global-close" :class="{hidden: $route.name === `export-file-success`}" @click="close">Back to {{ request.appName }}</a> -->
-    </div>
-</template>
-
 <script lang="ts">
-import {Component, Emit, Watch, Vue} from 'vue-property-decorator';
-import {SmallPage} from '@nimiq/vue-components';
-import {ParsedExportFileRequest, ExportFileResult} from '../lib/RequestTypes';
-import {State} from 'vuex-class';
+import { Component, Emit, Vue } from 'vue-property-decorator';
+import { ParsedExportFileRequest } from '../lib/RequestTypes';
+import { State } from 'vuex-class';
 import RpcApi from '../lib/RpcApi';
-import {ExportFileRequest, ExportFileResult as KExportFileResult} from '@nimiq/keyguard-client';
-import {State as RpcState, ResponseStatus} from '@nimiq/rpc';
+import { ExportFileRequest } from '@nimiq/keyguard-client';
+import { State as RpcState, ResponseStatus } from '@nimiq/rpc';
 import { KeyStore } from '@/lib/KeyStore';
 import staticStore, {Static} from '../lib/StaticStore';
 
-@Component({components: {SmallPage}})
+@Component({components: {}})
```

Comment:
I am not 100% sure, but I think this can be shortened to `@Component()` or `@Component({})`.

Reply:
Indeed it does.